packages/commonwealth/client/scripts/controllers/chain/cosmos/chain.ts

```diff
@@ -23,8 +22,10 @@ import {
   ITXData,
   ITXModalData,
 } from '../../../models/interfaces';
-import WebWalletController from '../../app/web_wallets';
+import { ETHERMINT_CHAINS } from '../../app/webWallets/keplr_ethereum_web_wallet';
```

Comment:
To be totally clear, I don't think injective uses ethermint proper -- I think they use another implementation of EVM on Cosmos. I could be wrong though + this variable is fine, as long as we document.

Reply:
Yeah maybe we need a better name here. I do think they are both ethermint-based, but maybe a better name is COSMOS_EVM_CHAINS.